kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -42,8 +42,8 @@ import ShowFor from "sumo/js/showfor";
       // Submit form
       $('#id_comment').keypress(function(e) {
         if (e.which === 13) {
-          $(this).blur();
-          $(this).closest('form').find('[type=submit]').click();
+          $(this).trigger('blur');
+          $(this).closest('form').find('[type=submit]').trigger('submit');
```

Comment:
I think this should be `'click'` instead of `'submit'`?
```suggestion
          $(this).closest('form').find('[type=submit]').trigger('click');
```